### PR TITLE
Add new DSLs PUB and PUB PROTO for model-based publishing.

### PR DESCRIPTION
**Pull request by oliver-reinhard**
@johanwannheden 
Johan, I added the **pom.xml** files to all the new bundles and adapted them as discussed. However, I think I'm supposed to also add the bundles to some **pom.xml** in **releng** but I couldn't find one that had all the other projects listed also.

### CODE REVIEW

File: com.mimacom.ddd.pub.proto.feature/feature.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature id="com.mimacom.ddd.pub.proto.feature"
+	label="PubProto Feature "
+	version="1.0.0.qualifier">
```
Comment (johanwannheden) [inline com.mimacom.ddd.pub.proto.feature/feature.xml]: Version should be `0.1.0.qualifier`

File: com.mimacom.ddd.pub.pub.feature/feature.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature id="com.mimacom.ddd.pub.pub.feature"
+	label="Pub Feature "
+	version="1.0.0.qualifier">
```
Comment (johanwannheden) [inline com.mimacom.ddd.pub.pub.feature/feature.xml]: Should be 0.1.0.qualifier

File: com.mimacom.ddd.pub.proto.feature/feature.xml (hunk 1 above)
Comment (oliver-reinhard) [reply com.mimacom.ddd.pub.proto.feature/feature.xml]: Done.

File: com.mimacom.ddd.pub.pub.feature/feature.xml (hunk 2 above)
Comment (oliver-reinhard) [reply com.mimacom.ddd.pub.pub.feature/feature.xml]: Done.

**Pull request merged**